apps/astarte_appengine_api/lib/astarte_appengine_api/device/device.ex

```diff
@@ -991,22 +991,46 @@ defmodule Astarte.AppEngine.API.Device do
       count == 1 ->
         [only_path] = paths
 
-        retrieve_endpoint_values(
-          client,
-          device_id,
-          :object,
-          :datastream,
-          interface_row,
-          endpoint_id,
-          endpoint_row,
-          only_path,
-          opts
-        )
+        {:ok, %Astarte.AppEngine.API.Device.InterfaceValues{data: values, metadata: metadata}} =
```

Comment:
Use a with, otherwise you'll crash if `retrieve_endpoint_values` returns an error